service_mesh/v2x/prepare-to-deploy-applications-ossm.adoc

```diff
@@ -7,6 +7,11 @@ toc::[]
 
 When you deploy an application into the {ProductShortName}, there are several differences between the behavior of applications in the upstream community version of Istio and the behavior of applications within a {ProductName} installation.
 
+[NOTE]
+====
+Do not deploy applications to the same namespace as {ProductShortName}. The control plane (service mesh Operators, Istiod, and `ServiceMeshControlPlane`) and data plane (applications and Envoy proxy) must be in separate namespaces.
```

Comment:
"Do not deploy applications within the {ProductShortName} control plane namespace"?